src/Refactoring-Core/RBPushDownMethodRefactoring.class.st

```diff
@@ -31,7 +31,6 @@ RBPushDownMethodRefactoring class >> pushDown: selectorCollection from: aClass [
 RBPushDownMethodRefactoring >> preconditions [
 
 	| condition |
```

Comment:
Could you add comments?
Because I could not understand why the selectorForView got removed.

Reply:
Now the selection of methods is done from the command (in #prepareFullExecutionInContext :) and the PushDown refactoring class is only limited to receiving a list of selectors to refactor